modules/ROOT/pages/cap-policy.adoc

```diff
@@ -3,39 +3,42 @@ ifndef::env-site,env-github[]
-This policy checks TCP protocol message sizes and headers. This policy does not check content.
+Anypoint Security uses the HTTP Limits policy to prevent an attacker from sending large messages that can consume bandwidth and result performance degradation or system outage. This policy applies to all the APIs behind your Edge inbound endpoint. You can apply API Gateway policies to each API to enforce other API-specific policies, such as throttling and JSON threat protection.
 
-This policy applies to all the APIs behind your Edge inbound endpoint. You can apply API Gateway policies to each API to enforce other API specific policies, such as throttling and JSON threat protection.
+The HTTP Limits policy limts the size of TCP protocol messages, paths, headers, and footers. This policy does not perform checks on the content of a message.
 
-When you configure a DoS policy, violations to the HTTP Limits policy escalate as protocol errors.
+When you configure a DoS Policy, violations to the HTTP Limits policy escalate as protocol errors.
 
 
 [[anypoint_security_prereqs]]
 == Prerequisites
 
 include::partial$anypoint-security-policies-prereqs.adoc[]
 
-== Configure an HTTP Limit Policy
+== Configure an HTTP Limits Policy
 
 You can configure and apply an HTTP Limits policy to increase or decrease default HTTP limits. 
 
 . Navigate to *Anypoint Security*, click the *Create Policy* icon, and select *Content Attack Prevention*.
 . Add a name for your policy in the *Name* field.
-. Configure the maximum sizes for message, path header and trailers in the fields below:
+. Configure the following parameters for your policy:
 +
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Value |Description
-|*Maximum Message Size* a| The default maximum message size allowed is 104857600. If request body scanning is enabled, this value should be set no larger than required to prevent attackers from abusing request body checking and exhausting resources. +
-Modify the message size based on your application needs, and base the size limits on what all of your APIs can handle. 
-|*Maximum Path Length* | The default maximum path size allowed is 4096 bytes.
-|*Maximum Length Of a Single Header* | The default maximum header length allowed for is 16384 bytes.
-|*Maximum Length Of a Single Trailer* | The default maximum trailer length allowed is 16384 bytes.
-|*Maximum Number Of Headers and Trailers* | The default maximum number of headers and trailers allowed is 32 kB.
+|*Maximum Message Size* a| The default maximum message size allowed is 104857600 bytes. +
+Modify this value based on the requirements of your application by setting the maximum size to a value based on a size your APIs can handle. +
+If the request body scanning option is enabled, do not set this value to a value larger than required to prevent attackers from abusing request body checking and exhausting resources.
```

Comment:
Delete "If the request body scanning option is enabled,". This text makes sense in the context of the WAF policy but is confusing when it only applies to the Limits policy. The sentence seems OK without that leading prepositional phrase from a technical content point of view.